index.md

```diff
@@ -86,6 +86,13 @@ Follow these instructions which will guide you through the setup process:
 
 * [Setup your FeatureBase cloud account](/docs/cloud/cloud-home)
 
+## How do I launch FeatureBase with Docker?
+You can quickly start FeatureBase Community using the [Docker Hub](https://hub.docker.com/r/featurebasedb/featurebase) image:
+
+```
+docker run -p 10101:10101 featurebasedb/featurebase
```

Comment:
Some more questions:
* Where should I run the docker pull command? (I'm assuming from a `/featurebase` directory?)
* Do I **then** build FeatureBase from source in that directory
* And once FeatureBase is built from source, do I then run `docker run -p 10101:10101 featurebasedb/featurebase`?

Reply:
The docker pull command pulls from Docker Hub. There's no code build or download required. It's just paste and go.

To clarify, there is no pull from Github. The user, if they are running Docker, will start an instance of FeatureBase Community (binary) locally with the pull. They would need to know to open the URL for the UI, however: http://0.0.0.0:10101